### DISCUSSION TITLE
OpenHank 3.3.3-1 win

### DISCUSSION THREAD

**Discussion by Rodoz** [OpenHantek/OpenHantek6022#400] [Q&A] [answered]
After I plug in my 6022BE then run OpenHante.exe the red LED flashes , the loading window briefly appears requesting module type with the default being 6022BE. Immediately following this the OpenHantek oscilloscope window appears but it is black.  No image top line displays openhantek6022 (3.3.3) - Device DSO-6022BE (FW0210)       The display area is black but if I scroll the mouse pointer horizontally below the top banner line the pop up menus appear as I scroll across.  Has anyone got an ide why this is not running.  The LED light is green after startup.    Has anyone experienced this and resolved it ?  Regards Rodoz

**Comment by Ho-Ro** [comment, marked as answer]
You could try the MSVC build from the latest [devdrop](https://github.com/OpenHantek/OpenHantek6022/releases/download/devdrop/openhantek_3.4-rc2-7-gad43d3b-devdrop_msvc_x64.zip) or from [3.4-rc2](https://github.com/OpenHantek/OpenHantek6022/releases/download/3.4-rc2/openhantek_3.3.3-30-g1b56a17_msvc_x64.zip). On some windows systems the graphic fails when using the MINGW build, so I revived the MSVC. Both binaries are much more stable than the old 3.3.3 builds from last summer. As you can see, I'm getting closer to a new 3.4 release (if only I had enough time).
